platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/address/AddressBookNetworkUtils.java

```diff
@@ -37,6 +37,13 @@ public final class AddressBookNetworkUtils {
 
     private AddressBookNetworkUtils() {}
 
+    /**
+     * Check if the address is local to the machine.
+     *
+     * @param address the address to check
+     * @return true if the address is local to the machine, false otherwise
+     * @throws IllegalStateException if there is an error while checking if the address is local.
```

Comment:
The illegal state exception isn't thrown anymore

Reply:
My mind is still on vacation.  :-) removed.